modules/openapi-generator/src/main/resources/r/ApiResponse.mustache

```diff
@@ -45,14 +45,11 @@ ApiResponse <- R6::R6Class(
     #'
     #' @param from_encoding The encoding of the raw response.
     #' @param to_encoding The target encoding of the return value.
-    response_as_text = function(from_encoding = "", to_encoding = "UTF-8") {
+    ResponseAsText = function(from_encoding = "", to_encoding = "UTF-8") {
```

Comment:
shall we keep the snake case convention for the time being?

changing it results in breaking changes (without fallback) which is not something we want to do in the next minor release v7.11.0 (FYI we allow breaking changes *with* fallback in minor release)

Reply:
to use PascalCase, I can think of a solution using lambda but I prefer we deal with it in another PR instead.

Reply:
Sure, I'll revert to snake case. I switched to pearl based on the contributing guide, but I appreciate avoiding breaking changes.

Reply:
done